tables/TaQL/TaQLNodeHandler.cc

```diff
@@ -943,6 +942,17 @@ namespace casacore { //# NAMESPACE CASACORE - BEGIN
     }
   }
 
+  void TaQLNodeHandler::handleJoins (const TaQLMultiNode& node)
+  {
+    if (! node.isValid()) {
+      return;     // no joins
+    }
+    const std::vector<TaQLNode>& nodes = node.getMultiRep()->itsNodes;
+    for (uInt i=0; i<nodes.size(); ++i) {
```

Comment:
I guess a range based loop could be used here, but I would immediately agree if you prefer the current way for consistency within the file.

Reply:
You're right. It is used at many other places in the code.
I might change it one day.